docs-src/typescript/logging.md

```diff
@@ -229,6 +229,64 @@ Some important features of the [InjectedSinkFunction](https://typescript.tempora
 - **Limited arguments types**: The remaining Sink function arguments are copied between the sandbox and the Node.js environment using the [structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm).
 - **No return value**: To prevent breaking determinism, Sink functions cannot return values to the Workflow.
 
+#### Shared logger interface
```

Comment:
I think this might be targeting the legacy docs set?
https://legacy-documentation-sdks.temporal.io/typescript/logging

that content is now published through another repo -
Can add it here if you want but it won't show up published.

Reply:
Could you add Val to that repo? (or make public)

Reply:
It looks like I do have access to that repo, I'll make the changes there.